reference/halyard/high-availability.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: single
+title:  "High Availability"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+This page describes how you can configure a Halyard deployment to increase the availability of specific services beyond simply [horizontally scaling](/setup/productionize/scaling/horizontal-scaling/) the service. Halyard does this by splitting the functionalities of a service into separate logical services. The benefits of doing this is specific to the service that is being split.
```

Comment:
'logical services' -> 'logical roles'? The service/services terminology is a bit confusing.